### DISCUSSION TITLE
Add support for EndeavourOS

### DISCUSSION THREAD

**Discussion by slaecker** [antonym/netboot.xyz#691] [Ideas] [answered]
Please add support for EndeavourOS, an Arch based distro that provides unbranded desktops and so is a great alternative to Manjaro.

Latest ISO release is at [https://endeavouros.com/latest-release/](https://endeavouros.com/latest-release/)

**Comment by antonym** [comment, marked as answer]
EndeavourOS is up on the Live menu.
